client/hotrod-client/src/test/java/org/infinispan/client/hotrod/query/RemoteQueryStringTest.java

```diff
@@ -97,10 +97,14 @@ protected ModelFactory getModelFactory() {
       return cache;
    }
 
+   protected int getNodesCount() {
```

Comment:
I don't think this is used. The derived class overrides ```createCacheManagers``` anyway.

Reply:
I see, let me sort this out